webapp/sources/rudder/rudder-core/src/main/scala/com/normation/cfclerk/domain/VariableAndSectionSpec.scala

```diff
@@ -437,3 +436,20 @@ object DisplayPriority {
     }
   }
 }
+
+sealed  trait CompositionRule
```

Comment:
Ho, it's THAT rule, not the other one. 
I think we need to change that name, it is extremely misleading.
`GroupedComponentComplianceAlgo` ? Or if the algo can be used for other elements than for `GroupedComponents`, it can be:  `ComplianceComputingAlgo`, `ComplianceSumAlgo`, `ComplianceReduceAlgo`, ...